subgraphs/risk/data.js

```diff
@@ -0,0 +1,11 @@
+export const RISK = [
+  {
+    id: "user:1",
+    name: "User One",
```

Comment:
You don't need the name here. Your schema only knows about id and riskRating

```graphql
type User @key(fields: "id") {
  id: ID!
  riskRating: Int
}
```

Reply:
If you didn't want to hard code data we could just do a Math.random() in the resolver